doc/design/mpi.md

```diff
@@ -0,0 +1,58 @@
+## MPI Jobs in Kube-Batch
+
```

Comment:
Just wondering if it's possible to leverage kubeflow/mpi-operator?

Reply:
A lot of the logic is similar, but after talking with @k82cn we didn't want to implement this as a Statefulset (which iirc Kubeflow's MPI operator does)